init/eessi_archdetect.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# current pathways implemented in EESSI
+# x86_64/generic
+# x86_64/intel/haswell
+# x86_64/intel/skylake_avx512
+# x86_64/amd/zen2
+# x86_64/amd/zen3
+# aarch64/generic
```

Comment:
`graviton2` + `graviton3` is missing here, should also be covered by tests

Reply:
done